src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/TensorsData.cs

```diff
@@ -111,24 +163,16 @@ public void SetTensorData(int index, byte[] buffer)
         /// <param name="index">The index of the tensor.</param>
         /// <returns>Raw tensor data</returns>
         /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
-        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <exception cref="IndexOutOfRangeException">Thrown when the method failed due to an invalid parameter.</exception>
```

Comment:
I agreed with your opinion. Then, `CheckIndexBoundary()` of `TensorsInfo` class also raise `ArgumentException` instead of `IndexOutOfRangeException`. right?

Reply:
yes, right